apps/web/.eslintrc.json

```diff
@@ -15,6 +15,9 @@
   "parserOptions": {
     "project": "./tsconfig.json"
   },
+  "plugins": [
+    "react"
+  ],
```

Comment:
I thought `next/core-web-vitals` already included the `eslint-plugin-react`. Did you try adding the rule without introducing the plugin here? If rules weren't included, maybe we want to add `plugin:react/recommended` to `extends` instead of adding the plugin here that way we enforce recommended React patterns, like the rules of hooks.

Reply:
I didn't try, let me check it out. 👍

Reply:
Seems like it is included like you suggested, so I removed the standalone plugin install.